packages/lb-utils/src/PointCloudUtils.ts

```diff
@@ -515,16 +528,31 @@ class PointCloudUtils {
     extraBoxList: IPointCloudBox[];
     extraSphereList?: IPointCloudSphere[];
   }) {
-    let trackID = 1;
     const boxList = this.getAllPointCloudResult({ imgList, step, extraBoxList, extraSphereList });
 
-    boxList.forEach((data: IPointCloudBox | IPointCloudSphere) => {
-      if (typeof data?.trackID === 'number' && data.trackID >= trackID) {
-        trackID = data.trackID + 1;
+    const existingTrackIDs = boxList
+      .map((data) => data.trackID)
+      .filter((trackID): trackID is number => typeof trackID === 'number')
+      .sort((a, b) => a - b);
+
+    if (existingTrackIDs.length === 0) {
+      return 1;
+    }
+
+    let index = existingTrackIDs.length - 1;
+    while (index >= 0) {
+      const currentID = existingTrackIDs[index];
+
+      if (currentID && currentID < 1000 && !existingTrackIDs.includes(currentID + 1)) {
```

Comment:
why 1000？it seem that hard is not need

Reply:
The background of demand in the user community